stitches.config.ts

```diff
@@ -128,9 +128,9 @@ const stitches = createStitches({
     },
   },
   media: {
-    bp1: '(min-width: 520px)',
```

Comment:
I'm maybe wrong but I'm not seeing this change being related to the stitches update. I feel this change should be handled in another PR.